packages/react-core/src/demos/Tabs.md

```diff
@@ -437,3 +437,132 @@ TabsOpenWithSecondaryTabsDemo = () => {
+                  </TabContentBody>
+                </TabContent>
+                <TabContent key={11} eventKey={11} id={`tabContent${11}`} activeKey={activeNestedTabKey} hidden={11 !== activeNestedTabKey}>
+                  <TabContentBody>Control plane panel</TabContentBody>
+                </TabContent>
+                <TabContent key={12} eventKey={12} id={`tabContent${12}`} activeKey={activeNestedTabKey} hidden={12 !== activeNestedTabKey}>
+                  <TabContentBody>Operators panel</TabContentBody>
+                </TabContent>
+                <TabContent key={13} eventKey={13} id={`tabContent${13}`} activeKey={activeNestedTabKey} hidden={13 !== activeNestedTabKey}>
+                  <TabContentBody>Virtualization panel</TabContentBody>
+                </TabContent>
+              </FlexItem>
+            </Flex>
+          </CardBody>
+        </Card>
+      </GridItem>
+      <GridItem xl={4} md={6}>
+        <Flex direction={{ default: 'column' }} className="pf-u-h-100">
+          <FlexItem flex={{ default: 'flex_1' }}>
+            <Card isFullHeight>
+              <CardHeader>
+                <Title headingLevel="h3" >
+                  Title of Card
+                </Title>
+              </CardHeader>
+            </Card>
+          </FlexItem>
+          <FlexItem flex={{ default: 'flex_1' }}>
+            <Card isFullHeight>
+              <CardHeader>
+                <Title headingLevel="h3" >
+                  Title of Card
+                </Title>
+              </CardHeader>
+            </Card>
+          </FlexItem>
+        </Flex>
+      </GridItem>
+    </Grid>
+  );
+
+  return (
+    <DashboardWrapper hasNoBreadcrumb>
+      <PageSection isWidthLimited variant={PageSectionVariants.light}>
+        <Title headingLevel="h1" size="2xl">
+          Overview
+        </Title>
+      </PageSection>
+      <PageSection type="tabs" variant={PageSectionVariants.light} isWidthLimited>
+        <Tabs activeKey={activeTabKey} onSelect={handleTabClick} usePageInsets id="open-nested-tabs-example-tabs-list">
+          <Tab eventKey={0} title={<TabTitleText>Cluster 1</TabTitleText>} tabContentId={`tabContent${0}`} />
+          <Tab eventKey={1} title={<TabTitleText>Cluster 2</TabTitleText>} tabContentId={`tabContent${1}`} />
+        </Tabs>
+      </PageSection>
+      <PageSection isWidthLimited variant={PageSectionVariants.light}>
```

Comment:
I don't think you want a light page section variant here. If you remove it, it will make the background of the main section grey like the core demo

Reply:
Fixed, thanks!